wordpress-mobile/WordPress-Android
# Updated stories library commit hash

Fixes connected tests and local builds alike 🙃 (see https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Android/21409/workflows/08eab796-928f-47e9-bf88-878000d9ed26/jobs/96452)

See related PR https://github.com/Automattic/stories-android/pull/641

To test:
1. run `./gradlew assembleVanillaDebug`
2. verify it builds successfully

PR submission checklist:
